app/views/users/_form.html.erb

```diff
@@ -35,14 +41,14 @@
   <div class="form-group col-xs-12">
     <label class="col-xs-2 control-label" for="phone">Phone Number</label>
     <div class="col-xs-6">
-      <%= text_field :user, :phone, size: 20, placeholder: "111-222-3333", class:"form-control", required: true %>
+      <%= text_field :user, :phone, placeholder: "111-222-3333", class:"form-control", required: true %>
```

Comment:
This is the line I was referring to above, again would like to see this display XXX-XXX-XXXX